src/doc/alerts.xml

```diff
@@ -86,7 +86,8 @@ This list the alerts
 10046	Insecure Component
 10047	HTTPS Content Available via HTTP
 10048	ShellShock
-10049  Image Location Scanner - passive scanner finding
+10049   Image Location Scanner - passive scanner finding
```

Comment:
Yes, the id 10050 is in line 90. Additionally I removed a redundant space at line 89.